spec/oncocore_icd10_vs.txt

```diff
@@ -0,0 +1,2608 @@
+Grammar:	ValueSet 5.0
+Namespace:	oncocore
+
+CodeSystem:     SCT = http://snomed.info/sct
+CodeSystem:     ICD10CM = http://hl7.org/fhir/sid/icd-10-cm
+CodeSystem:     ICDO3MORPH = http://codes.iarc.fr/codegroup/2
+
+ValueSet:      AnyCancerDisorderCodeVS
+Description:   "A broad value set containing ICD10, ICD-O-3, and SNOMED CT codes for cancer conditions and morphologies."
```

Comment:
Is the `any` an attempt to say we have primary and secondary malignancies? We need to beef up the description on this one.

Reply:
Done.